practice_app/api/tests.py

```diff
@@ -101,4 +101,37 @@ def test_valid_rows(self):
         self.assertContains(response, 'source')
         self.assertContains(response, 'date')
         self.assertContains(response, 'url')
-        self.assertContains(response, 'position')
\ No newline at end of file
+        self.assertContains(response, 'position')
+class zenodo_test_cases(TestCase):
```

Comment:
This class naming doesn't quite align with our decision to use PEP-8 Style, see [class naming convention](https://peps.python.org/pep-0008/#class-names).

Reply:
Changed the class name in order to comply with our convention